tests/end-to-end/train.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+echo ""
+echo "Train "
+echo ""
+python $root_dir/main.py --train \
```

Comment:
we should still check for non zero exit status

Reply:
agree. removed it by accident. good catch!